sources/osgShadow/ShadowMap.js

```diff
@@ -930,14 +873,6 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
             //console.log( 'shadow early OUT' );
```

Comment:
Will rename as bbox 

( The bbox is used in as the frustumBounds, doesn't inmply it's a frustum, but that was when I was doing ops on it in that function, which not the case anymore)